duvet-core/src/artifact.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::path::Path;
+
+/// Synchronizes a value to the file system
+///
+/// When the `CI` environment variable is set, this method ensures the value matches
```

Comment:
```suggestion
/// When the `CI` environment variable is set, this method asserts the value matches
```

Reply:
nit: `ensures` sounds more like it changes what is on disk to match, rather than just fail if it doesn't match

Reply:
Yeah that makes sense